test/end-to-end/metrics.test.ts

```diff
@@ -104,7 +99,7 @@ async function getMetrics(): Promise<ParsedMetrics | undefined> {
 
 const projectRootPath = process.cwd();
 
-describe('Metrics', function () {
+describe('Metrics', { skip: isTestFiltered(['flaky']) }, function () {
```

Comment:
@djwhitt skipping this one. Tests were passing because they were not asserting anything with the if blocks.